docs/site/Sequence.md

```diff
@@ -83,14 +77,21 @@ class MySequence extends DefaultSequence {
 ## Custom Sequences
 
 Most use cases can be accomplished with `DefaultSequence` or by slightly
-customizing it:
+customizing it. When an app is generated by the command `lb4 app`, a sequence
+file extending `DefaultSequence` is already generated and bound for you so that
+easily customize it.
```

Comment:
Can you mention where a user can find this extended version of the DefaultSequence (The file name).